elements/search/nuxeo-saved-search-actions.js

```diff
@@ -101,9 +101,16 @@ Polymer({
       observer: '_searchFormChanged',
     },
     _dirty: Boolean,
+    _isSearchFormVisible: Boolean,
   },
 
-  _searchFormChanged() {
+  ready() {
+    this._searchFormVisibilityChanged = (event) => {
+      this._isSearchFormVisible = event.target.visible;
+    };
+  },
```

Comment:
Can we get away with not having to add a `ready` method just for this or is it strictly necessary to do it like this? (could probably mess up the `this` inside?)
```suggestion
  _searchFormVisibilityChanged(event) {
    this._isSearchFormVisible = event.target.visible;
  }
```

Reply:
Ah yes, you are absolutely correct. I tried that, but the problem is the `this` inside. With that we would end up having to use the `bind(this)` which would create a new function. 
That said, my first approach was to define the method inside the `searchFormChanged`, wdyt?